0800-appendix-198215-dale.Rmd

```diff
@@ -98,6 +100,8 @@ pH was `r fpr::fpr_my_habitat_info(loc = "ds", col_pull = 'p_h')` and
 conductivity was
 `r fpr::fpr_my_habitat_info(loc = "ds", col_pull = 'conductivity_m_s_cm')`uS/cm.
 
+The site was revisited in 2023, one culvert had very little flow and the other had moderate flow. There is a significant outlet drop (~1m) and deep outlet pool on the culvert with moderate flow, which is likely a barrier to upstream fish passage for certain life stages (Figure \@ref(fig:photo-198215-09)). 
```

Comment:
- tense switches around here
- it disucsses the outlet drop as though perhaps it changed.  I don't think it did so this type of value adding can be done in the original text. 
- There is often lots of help that can be given to almost every section of text when a memo is revisited. Since we are in there I would expect us to find other errors/sub-par writing and correct them and help tell the story better and better each time.  Read it as though you are the consumer of the information and try to make it better so it is more succinct and higher quality.

For example I would perhaps change "This crossing is the responsibility of the Ministry of Transportation and Infrastructure. The chris_culvert_id of this structure is 1524775." to read "This crossing is the responsibility of the Ministry of Transportation and Infrastructure (chris_culvert_id = 1524775)."  Often times too we refer to the number of the crossing but using the road name is commonly easier for the reader to understand.

Reply:
okay yup that all sounds good